docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -367,12 +367,24 @@ statements, the connector supports the following features:
 - {doc}`/sql/create-table`, see also {ref}`delta-lake-sql-basic-usage`
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
-- {doc}`/sql/alter-table`
+- {doc}`/sql/alter-table`, see also {ref}`delta-lake-alter-table`
 - {doc}`/sql/drop-schema`
 - {doc}`/sql/show-create-schema`
 - {doc}`/sql/show-create-table`
 - {doc}`/sql/comment`
 
+
+(delta-lake-alter-table)=
+
+### ALTER TABLE statement limitations
+
+The connector only supports the `ALTER TABLE RENAME TO` statement when met with
+the following conditions:
```

Comment:
this is misleading. .. from what I understand it is one of the the following conditions .. not both ..